src/com/twitter/intellij/pants/bsp/FastpassUtils.java

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.bsp;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.twitter.intellij.pants.util.PantsUtil;
+import org.apache.commons.io.IOUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+final public class FastpassUtils {
+
+  @NotNull
+  public static Stream<VirtualFile> pantsRoots(Module module) {
+    return Stream.of(ModuleRootManager.getInstance(module).getSourceRoots()).flatMap (
+      sourceRoot -> toStream(PantsUtil.findPantsExecutable(sourceRoot.getPath()).map(VirtualFile::getParent))
+    );
+  }
+
+  public static CompletableFuture<Void> amendAll(@NotNull PantsBspData importData, Collection<String> newTargets) throws IOException {
+    List<String> amendPart = Arrays.asList(
+      "amend", importData.getBspPath().getFileName().toString(),
+      "--addresses-list", String.join(",", newTargets)
```

Comment:
We should include `--no-bloop-exit` to ensure that Fastpass doesn't kill the bloop server and also `--intellij` so that the fastpass command doesn't trigger `./pants export-dep-as-jar` (we want to do that as part of "refresh BSP projects" in IntelliJ)
```suggestion
      "--no-bloop-exit", "--intellij", "--intellij-launcher", "echo",
      "--new-targets", String.join(",", newTargets)
```